app/src/main/java/com/imashnake/animite/features/searchbar/SearchFrontDrop.kt

```diff
@@ -177,9 +215,7 @@ fun ExpandedSearchBar(viewModel: SearchViewModel = viewModel()) {
                 IconButton(
                     onClick = {
                         text = ""
```

Comment:
Does text not update based on VM state automatically? Maybe we can do something about that